FirebaseAuth/Tests/Sample/AuthSample.xcodeproj/project.pbxproj

```diff
@@ -7,7 +7,11 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		01FFDAAA77DA16EB3FF8D384 /* Pods_Auth_ApiTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = EF364171DEE37363A6997CC7 /* Pods_Auth_ApiTests.framework */; };
```

Comment:
We check in `pod deintegrate` versions of Xcode projects and I'm not sure this file is needed.

Reply:
Gotcha. Fixing now.